apps/cowswap-frontend/src/legacy/state/price/updater.ts

```diff
@@ -22,13 +23,14 @@ import { useWalletInfo } from 'modules/wallet'
 import { getPriceQuality } from 'api/gnosisProtocol/api'
 import { LegacyFeeQuoteParams as LegacyFeeQuoteParamsFull } from 'api/gnosisProtocol/legacy/types'
 import { useVerifiedQuotesEnabled } from 'common/hooks/featureFlags/useVerifiedQuotesEnabled'
+import { useSafeEffect } from 'common/hooks/useSafeMemo'
 import tryParseCurrencyAmount from 'lib/utils/tryParseCurrencyAmount'
 
 import { useAllQuotes, useIsBestQuoteLoading, useSetQuoteError } from './hooks'
 import { QuoteInformationObject } from './reducer'
 
 export const TYPED_VALUE_DEBOUNCE_TIME = 350
-const REFETCH_CHECK_INTERVAL = 10000 // Every 10s
+const REFETCH_CHECK_INTERVAL = 30000 // Every 30s
```

Comment:
Don't re-fetch the price too often. 
30s should be good enough and would give a break to the API.

Reply:
`wasQuoteCheckedRecently` check looks irrelevant now. Since we fetch only every 30sec, it means the condition always should be false

Reply:
The issue is that the hooks calls eagerly to `refetchQuoteIfRequired` so a re-render can also make it quote faster. `wasQuoteCheckedRecently` is just a protection

anyways, I'm re-applyign the changes in DEV as the rebase is too chaotic